daemon/policy.go

```diff
@@ -232,6 +232,12 @@ func (d *Daemon) policyAdd(rules policyAPI.Rules, opts *AddOptions) (uint64, err
 func (d *Daemon) PolicyAdd(rules policyAPI.Rules, opts *AddOptions) (uint64, error) {
 	log.WithField(logfields.CiliumNetworkPolicy, logfields.Repr(rules)).Debug("Policy Add Request")
 
+	fqdnMarkedRules, err := d.dnsPoller.StartPollForDNSName(rules)
```

Comment:
Do we need to undo anything in the error patch below? If not, please add a comment why not.

Reply:
yes, we do/should. I'll fix it.